app/views/components/_result-card.html.erb

```diff
@@ -0,0 +1,22 @@
+<%
+  title ||= nil
+  description ||= nil
+  url ||= nil
+  url_text ||= nil
+%>
+<div class="app-c-result-card">
+  <%= render "govuk_publishing_components/components/heading", {
+    text: title,
+    heading_level: 2,
```

Comment:
Are we sure that this is only going to be needed as a level 2 heading? If so, fine - otherwise it'd be worth looking into whether the [shared helper from the components gem][1] could be used:

```erb
shared_helper = GovukPublishingComponents::Presenters::SharedHelper.new(local_assigns)
```
```suggestion
    heading_level: shared_helper.get_heading_level,
```

The shared helper defaults to a level two heading, so this wouldn't change the output unless the `heading_level` parameter was set.

[1]: https://github.com/alphagov/govuk_publishing_components/blob/99938773aef7a14e8f67c878f5ed0d9deddd09e0/lib/govuk_publishing_components/presenters/shared_helper.rb#L28-L32